tests/foreign_object/models/customers.py

```diff
@@ -39,3 +40,27 @@ class Contact(models.Model):
         to_fields=["customer_id", "company"],
         from_fields=["customer_code", "company_code"],
     )
+
+
+class Patron(models.Model):
+    pass
```

Comment:
Can we rename PatronTab -> CustomerTab and target `Customer` to avoid creating this table?